Multisig/Data/Services/Utils/DelegateWarningCalculator.swift

```diff
@@ -0,0 +1,47 @@
+//
+// Created by Dirk Jäckel on 08.06.22.
+// Copyright (c) 2022 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+// used to enhance txData to contain trustedDelegateCallTarget flags in every displayed tx (also multisend inner txs)
+class DelegateWarningCalculator {
+    static func addMissingTrustedDelegateCallTargets(txData: inout SCGModels.TxData) {
+
+        // TODO check: if operation is DELEGATE then to-address must be in addressInfoIndex otherwise trustedDelegateCallTarget = false
+        // if txData.to
+
+        // for testing. This should replace any false value with true ad thus fail the first test
+        txData.trustedDelegateCallTarget = true
```

Comment:
This is supposed to modify the parameter (`txData`) but as you can see in the test result, ist doesn't. Same for the other modifications later in this method.

Reply:
this is modified OK